src/components/ScholarshipAmountField.jsx

```diff
@@ -1,20 +1,21 @@
 import React from 'react';
 import {
-  FormControl,
-  InputLabel,
-  Select,
-  MenuItem,
   TextField,
+  FormLabel,
+  Radio,
+  RadioGroup,
+  FormControlLabel,
+  Grid,
 } from '@material-ui/core';
 import { makeStyles } from '@material-ui/core/styles';
 import PropTypes from 'prop-types';
 import AmountType from '../types/AmountType';
 
-const useStyles = makeStyles({
+const useStyles = makeStyles((theme) => ({
   formControl: {
-    minWidth: 200,
+    margin: theme.spacing(1),
```

Comment:
This causes the margin on the left to be off. I recommend setting the `spacing` property in the `Grid container` below instead.